src/startupscriptgenerator/node/main.go

```diff
@@ -39,6 +39,17 @@ func main() {
 
 	common.SetGlobalOperationId(fullAppPath)
 
+	buildManifest := common.GetBuildManifest(fullAppPath)
```

Comment:
@gpcastro As discussed earlier, we check the manifest to see if the output is compressed. If yes, we un-compress it using a separate script and then call into other logic which remains unchanged.